src/atoms/Trackable/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { dataAttributes } from './utils';
+
+function wrappedChild(children, data) {
+  const child = React.Children.only(children);
```

Comment:
The only question I have here is, what is the error that gets thrown? Is it possible to provide a clearer error?